app/connectors/BankAccountInsightsHttpParser.scala

```diff
@@ -34,12 +35,16 @@ object BankAccountInsightsHttpParser {
             case JsSuccess(model, _) =>
               Right(model)
 
-            case _ =>
+            case _ => {
+              logger.warn(s"Unable to parse the content of a response from Bank Account Insights")
```

Comment:
can we add anything to identify the user? do you know if claims has the session Id at least?

Reply:
I'm a bit concerned about putting those in the logs.

Reply:
As we spoke, I think you meant request id, so I added request id on the ConnectorFailureLogger extension.